### PR TITLE
Input group new topic

### PR DESCRIPTION
**Pull request by zdrawku**
Changing the PR to be from upstream/branchName and not tiliev/branchName

Added IgxInputGroup topic.
Added translations for new input group topic
Adding keywords and requesting clarification

### CODE REVIEW

File: en/components/input_group.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+
+```html
+<igx-input-group>
+    <input igxInput name="fullName" type="text" [(ngModel)]="user.fullName" required="required" />
+    <label igxLabel for="fullName">Full Name</label>
+</igx-input-group>
+```
+
+### Prefix & Suffix
+If you want to have an input prefix or suffix, you can use Ignite UI for Angular Prefix or Suffix. [Please explain what the Prefix and Suffix can contain] Let's add a new input field with prefix and suffix:
```
Comment (kdinev) [inline en/components/input_group.md]: Address the comment left in the doc

**Comment by kdinev** [comment]
Assigning back to @ig-ealbert to add the newly added changes to the translation.

File: jp/components/input_group.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
+﻿---
+title: Input Group コンポーネント
+_description: Ignite UI for Angular Input Group コンポーネントを使用すると、単一行または複数行のテキスト要素を作成し、CSS スタイルを追加し、その他のコントロールと統合できます。
+_keywords: Ignite UI for Angular, UI コントロール, Angular ウィジェット, web ウィジェット, UI ウィジェット, Angular, ネイティブ Angular コンポーネント スィート, ネイティブ Angular コントロール, ネイティブ Angular コンポーネント ライブラリ, Angular Label コンポーネント, Angular Label コントロール, Angular Input components, Angular Input controls, Input component, Input control, Label component, Label control, Angular Input Group components, Angular Input Group controls, Angular Input directive, Angular Label directive, Angular Forms, Angular Reactive Forms, Angular Form Validation
```
Comment (ig-ealbert) [inline jp/components/input_group.md]: Angular Input コンポーネント, Angular Input コントロール, Input コンポーネント, Input コントロール, Label コンポーネント, Label コントロール, Angular Input Group コンポーネント, Angular Input Group コントロール, Angular Input ディレクティブ, Angular Label ディレクティブ, Angular Forms, Angular Reactive Forms, Angular Form 検証

Comment (kdinev) [reply jp/components/input_group.md]: @ig-ealbert The branch is in the Ignite UI remote. Remove the old fork and clone again. You should be able to switch to it.

**Comment by ig-ealbert** [comment]
I'll add the translation for this new sentence tomorrow, once it is reviewed.  I will translate it today, along with the other new content that has been added in other PRs.

File: en/components/input_group.md [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+
+```html
+<igx-input-group>
+    <input igxInput name="fullName" type="text" [(ngModel)]="user.fullName" required="required" />
+    <label igxLabel for="fullName">Full Name</label>
+</igx-input-group>
+```
+
+### Prefix & Suffix
+If you want to have an input prefix or suffix, you can use Ignite UI for Angular Prefix or Suffix. Both directives can contain html elements, strings or event other components. Let's add a new input field with prefix and suffix:
```
Comment (ig-ealbert) [inline en/components/input_group.md]: @zdrawku "even", not "event".

**Comment by ig-ealbert** [comment]
I added a comment about a typo which I noticed while translating this new sentence.

**Comment by kdinev** [comment]
@ig-ealbert I'm good with the topic. It's your turn now to translate the latest changes. :)

**Comment by zdrawku** [comment]
@ig-ealbert, is there something else from your side as comments or implementation?

**Pull request merged**